package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@viamrobotics/prime",
-  "version": "0.0.88",
+  "version": "0.0.89",
```

Comment:
probably have to bump the version one more time, I think someone just pushed a new version at `0.0.89`